deepchem/models/tensorgraph/tests/test_layers_eager.py

```diff
@@ -0,0 +1,980 @@
+import deepchem as dc
+import numpy as np
+import tensorflow as tf
+import deepchem.models.tensorgraph.layers as layers
+import tensorflow.contrib.eager as tfe
+from tensorflow.python.eager import context
+from tensorflow.python.framework import test_util
+import rdkit
+
+
+class TestLayersEager(test_util.TensorFlowTestCase):
+  """
+  Test that layers function in eager mode.
+  """
+
+  def test_conv_1d(self):
+    """Test invoking Conv1D in eager mode."""
+    with context.eager_mode():
+      with tfe.IsolateTest():
+        width = 5
+        in_channels = 2
+        filters = 3
+        kernel_size = 2
+        batch_size = 10
+        input = np.random.rand(batch_size, width, in_channels).astype(
```

Comment:
nit: input is a python keyword

Reply:
I don't think so?  It's not listed as one at https://www.programiz.com/python-programming/keywords-identifier.

Reply:
It is a python built-in function not a keyword.  using it as a variable overwrites the variable from the built-in function to the variable but it does keep the weird highlighting by text editors as if it were a function as seen here.

We don't have to change it just a comment.